CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ Features:
 
 Bug Fixes:
 - When using CMake presets, the Project Status View now shows the build target along with the build preset. [PR #3241](https://github.com/microsoft/vscode-cmake-tools/pull/3241)
+- Fix per-folder browse configurations returning incorrect information [#3155](https://github.com/microsoft/vscode-cmake-tools/issues/3155)
```

Comment:
Our cpptools changelog always end changelog lines with a period, i.e. before the markdown link. I see that style followed in the line above but not below, so I'm not sure if you guys follow that same style or not.

Reply:
Updated.